client/layout/masterbar/style.scss

```
@@ -565,6 +570,12 @@ body.is-mobile-app-view {
 
 	.is-support-session &.is-active {
 		background: var(--studio-orange-70);
+		--color-masterbar-icon: var(--studio-gray-5) !important;
+		--color-masterbar-text: var(--studio-gray-5) !important;
+	}
+
+	.is-support-session &:hover {
+		background: var(--studio-orange-50);
 	}
 
```

The tab background doesn't change on hover on an active tab on the regular bar. Here's my attempt at doing that for the support session bar:

```suggestion
.is-support-session &.is-active:hover {
		background: var(--studio-orange-70);
		color: var(--studio-gray-5);
	}
```

https://github.com/user-attachments/assets/1876e5a2-2c9a-4da3-a1c5-e267bf056001

---

Updated [here](https://github.com/Automattic/wp-calypso/tree/fc3a63ca7a91a2605c69ec7e71e6f507edd2af5f), thanks for the suggestion, it works great :tada: